Makefile

```diff
@@ -1,4 +1,18 @@
 
+.PHONY: default
+default:
+	@echo "Desk doesn't need to be compiled. Please run \"sudo make install\" to install desk." ||:
```

Comment:
Suppresses make's erroring if the command fails. Not needed on the echo lines actually so feel free to take them out. See [my comment](https://github.com/jamesob/desk/pull/24#issuecomment-153400441) on the PR for the full explanation